apps/api-server/src/models/Resource.js

```diff
@@ -1025,12 +1052,14 @@ module.exports = function (db, sequelize, DataTypes) {
       // published
       if (!self.publishDate) return false;
       // status
-      let status = self.statuses?.find((tag) => tag.type === 'status');
-      if (
-        typeof status?.extraFunctionality?.noComment == 'boolean' &&
-        status.extraFunctionality.noComment
-      ) {
-        return false;
+      let statuses = self.statuses || [];
+      for (let status of statuses) { // todo: wil je dat zo?
+        if (
+          typeof status?.extraFunctionality?.noComment != 'boolean' ||
```

Comment:
Maar wat als de waarde van .noComment false is? dan is het wel een boolean en returned die true